public/chore.js

```diff
@@ -0,0 +1,32 @@
+function clearFields() {
+    const elements = document.getElementById('chore-form').elements;
+
+    for (i = 0; i < elements.length; i++) {
+        fieldType = elements[i].type.toLowerCase();
+        switch (fieldType) {
+        case 'text':
+            elements[i].value = '';
+            break;
+        case 'checkbox':
+            if (elements[i].checked) {
+                elements[i].checked = false;
+            }
+            break
+        default:
+            break;
+        }
+    }
+}
+
+
+const choreList = ["Make Bed", "Walk Dogs", "Clean Room", "Empty Dishwasher"];
+const list = document.createElement('ul');
+
+choreList.forEach(function (choreList) {
+    const li = document.createElement('li');
+    li.textContent = choreList;
```

Comment:
Now that we are iterating through objects, we will need to set the textContent to the object's properties.

If we just wanted the chore to be listed like you had before, we would do:
`li.textConent = chore.name`

BUT I would like to see the it look like this:
- Chore: Make Bed - Assigned to: Kyle

So we are going to use [_template literals_](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) like this:
```
li.textConent = `Chore: ${chore.name} - Assigned to: ${chore.assignee}`
```